pkgs/desktops/gnome/extensions/update-extensions.py

```diff
@@ -37,6 +36,10 @@
 for shell_version in supported_versions.keys():
     package_name_registry[shell_version] = {}
 
+dir_path = Path(__file__).resolve().parent
+
+nix_store_pattern = re.compile(r"/nix/store/[\w|\.|-]*")
```

Comment:
I have 628 store paths not matching this regex:

```
python3 -c 'import re, glob; nix_store_pattern = re.compile(r"/nix/store/[\w|\.|-]*$"); print(list(filter(lambda p: not nix_store_pattern.match(p), glob.glob("/nix/store/*"))))'
```

Reply:
Good find, I've added a `+` and `=`. I'm not fixing the `.drv` ones, since those are anyway no store paths created by `nix-prefetch-url`

```
python3 -c 'import re, glob; nix_store_pattern = re.compile(r"/nix/store/[\w|\.|=|+|-]*$"); print(list(filter(lambda p: not nix_store_pattern.match(p) and not ".drv" in p, glob.glob("/nix/store/*"))))'
```